please_wait.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Unidling, please wait</title>
+  </head>
+  <body>
+    <h1>Unidling, please wait a few seconds</h1>
+    <script>
+(function () {
+
+  var delay = 3000;
```

Comment:
As these two variables are kind of "constants" you may want to capitalise them.

Reply:
Technically, these are not constants, because they are mutable. If they were `const`s, then sure, `const DELAY = 3000.`